src/bionics.cpp

```diff
@@ -360,6 +364,7 @@ bool player::activate_bionic( int b, bool eff_only )
         if( one_in( 5 ) ) {
             add_effect( effect_teleglow, rng( 5_minutes, 40_minutes ) );
         }
+        remove_effect( effect_supercharged ); // no cheese for you
```

Comment:
Remove the nonsense comment.
```suggestion
        remove_effect( effect_supercharged );
```